cmd/sonobuoy/app/status.go

```diff
@@ -165,37 +169,50 @@ func printAll(w io.Writer, status *aggregation.Status) error {
 
 func printSummary(w io.Writer, status *aggregation.Status) error {
 	tw := defaultTabWriter(w)
-	totals := map[string]map[string]int{}
+	totals := map[string]map[string]map[string]int{}
```

Comment:
I already had thought this was hard to follow; can we avoid adding the extra level of map? I'd like to not show each plugins duration either, just the max of the values in the group. If someone adds the `--show-all` then we would show duration of each.

Speaking of `--show-all`, that doesn't hav duration at all which it should now.
![image](https://user-images.githubusercontent.com/10273533/67497351-75bbe280-f643-11e9-9a1f-d53e6e8dea23.png)